components/infobox/wikis/stormgate/infobox_building_custom.lua

```diff
@@ -148,13 +148,43 @@ end
 ---@param args table
 ---@return string?
 function CustomBuilding:subHeaderDisplay(args)
-	if Logic.isEmpty(args.subfaction) or
-		string.find(args.subfaction, '1v1') or
-		string.find(args.subfaction, self.pagename) then return end
-	return tostring(mw.html.create('span')
+    local GAME_MODE_NAME = {
+		coop = 'Co-op',
+		mayhem = 'Team Mayhem'
+	}
+    local subfactionData = Array.parseCommaSeparatedString(args.subfaction)
+
+	if Table.includes(subfactionData, '1v1') then return tostring(mw.html.create('span')
 		:css('font-size', '90%')
-		:wikitext('Hero: ' .. self:_displayCsvAsPageCsv(args.subfaction))
-	)
+		:wikitext(Abbreviation.make('Standard', 'This is part of Head to Head 1v1. '
+			.. 'It might also be part of certain Hero rosters in Team Mayhem or Co-op.'))
+	) end
```

Comment:
```lua
	if Table.includes(subfactionData, '1v1') then
		return tostring(mw.html.create('span')
			:css('font-size', '90%')
			:wikitext(Abbreviation.make('Standard', 'This is part of Head to Head 1v1. '
				.. 'It might also be part of certain Hero rosters in Team Mayhem or Co-op.'))
		)
	end
```

better readable